packages/adapters/gatewallet/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { isInMobileBrowser } from '@tronweb3/tronwallet-abstract-adapter';
+
+export function supportGateWallet() {
+    return !!(window.gatewallet && window.gatewallet.tronLink);
+}
+
+export const isGateApp = /GateApp/i.test(navigator.userAgent);
+
+export function openGateWallet() {
+    if (!isGateApp && isInMobileBrowser()) {
+        window.location.href =
```

Comment:
It seems that this deeplink format can not open the dapp in gatewallet ios mobile.